src/ui-lib/sass/05-components/02-molecules/01-cards/_location-card.scss

```diff
@@ -0,0 +1,47 @@
+.grav-c-location-card {
+  position: relative;
+  margin: 0;
+  overflow: hidden;
+}
+
+.grav-c-location-card__content {
```

Comment:
I've made full use of nesting in situations like these, so I don't know if you split elements styles aside or if it's just a question because of the limit of nesting depth...

Reply:
You mean `&__content`? I prefer that but used to people hating it so didn't do it.